.github/workflows/integration_test.yaml

```diff
@@ -5,18 +5,12 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
+    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@add_support_for_canonical_k8s
```

Comment:
Would there be a PR for this as well so that we can get it pointing to `main`? :D

Reply:
Yep, we still need to have a discussion internally on how to properly implement this on main, since there's also an option of adding support for canonical k8s directly in `charmed-kuberentes/action-operator` in which case this patch won't be needed anymore

Reply:
So I'll leave it as it is for now :D